bin/gwdetchar-slow-correlation

```diff
@@ -295,6 +300,7 @@ def process_channel(input_,):
 
 pool = multiprocessing.Pool(nprocplot)
 results = pool.map(process_channel, auxdata.iteritems())
+#results = pool.map(partial(process_channel, thresh=threshold), auxdata.iteritems())
```

Comment:
it's just 06b0490d1867614d8be3683fa2b14792961441da that is causing problem, I think, you should be able to execute

```
git revert 06b0490d1867614d8be3683fa2b14792961441da
```

to remove it, then push.